src/main/java/com/twilio/sdk/resource/instance/Account.java

```diff
@@ -846,6 +847,181 @@ public UsageRecordList getUsageRecords() {
 	}
 
 	/**
+	 * Returns multiple Usage Records for each usage category,
+	 * each representing usage over a daily time-interval
+	 *
+	 * @return list of Usage Records over a daily time-interval
+	 */
+	public UsageRecordList getDailyUsageRecords() {
```

Comment:
yeah I agree with it. its a lot of duplicated code. It makes sense to replace them all with only two methods.